frontend/src/lib/widgetTheme.ts

```diff
@@ -239,6 +239,14 @@ export const buttonOverrides = {
   },
 }
 
+export const multiSelectOverrides = {
+  ValueContainer: {
+    style: () => ({
+      minHeight: "44px", // Fixing minimum height in order to prevent some changing height
```

Comment:
Hmm... this does fix it, but it feels a bit hacky.

Any idea what is causing BaseWeb to misbehave? I can't repro the bug in the BaseWeb page https://v8-17-1.baseweb.design/components/select/ , but when I remove all theming from widgetTheme.ts, I can still repro the bug. So my guess is our other styles are interfering with BaseWeb. I just can't figure out which styles are to blame.

It would be nice to figure out the source of this bug, so it can be fixed in a more future-proof manner.

If you spend a couple of hours on this and still can't find the underlying cause, feel free to merge this version. But please add a comment explaining whatever you found out, and link it to the original bug.

Reply:
I've tested it using the example from their documentation and it's doing the same, look at this:

<img width="710" alt="Screen Shot 2019-12-03 at 4 31 51 PM" src="https://user-images.githubusercontent.com/631997/70083071-84a29700-15ea-11ea-92c4-c10c9bd835bf.png">
<img width="628" alt="Screen Shot 2019-12-03 at 4 31 59 PM" src="https://user-images.githubusercontent.com/631997/70083085-8704f100-15ea-11ea-864f-f96535668024.png">